### ISSUE TITLE
[Angular][BackEnd] Create (add) new case - descoped [2]

### ISSUE THREAD

**Issue by laurentiustanvitagroup** [hzi-braunschweig/SORMAS-Project#5482]
### Situation Description

In view of the future replacement of the current FE with the Angular component, the Angular team will focus on the required changes.

### Feature Description

1. Create an endpoint for the "create new case"
2. The endpoint should return the saved object

### Possible Alternatives

N/A

### Additional Information

N/A

**Comment by MateStrysewske** [comment]
@laurentiustanvitagroup Is a new endpoint actually needed? Any reason to not use the existing /push endpoint (postCases)?

**Comment by marius2301** [comment]
A person can have several cases, thus there is a 1:n relation between person and cases.  
Currently the creation of the case is done in a separate call AFTER the case person is created (or selected). It means: 

- that when creating a case the user should have the possibility to select a person 
- the casedatadto, which is the payload for creating a case, contains only a casepersonereferencedto ( a uuid, firstname and lastname) of a person => the call to create a new person should be done before the call to create a case.

**Comment by marius2301** [comment]
Mandatory fields are:

- disease
- PersonReferenceDto caseperson(uuid)
- reportDate
- UserReferenceDto reportingUser (uuid)
- caseClassification
- investigationStatus
- FacilityReferenceDto healthFacility
- RegionReferenceDto region
- DistrictReferenceDto district

Also as mentioned before:
 -  the UUID of a new entity needs to be generated by the backend 
 - the personereferencedto it means that the case person is created before
